src/components/DonateButton.jsx

```diff
@@ -482,31 +483,24 @@ class DonateButton extends React.Component {
       buttons: ['Cancel', 'Lets do it!'],
     });
 
-    let result = false;
     if (isConfirmed) {
       try {
-        if (
-          await this.donateWithBridge(
-            dacId,
-            amountDAC,
-            donationOwnerAddress,
-            amount,
-            comment,
-            allowanceApprovalType,
-          )
-        )
-          result = await this.donateWithBridge(
-            adminId,
-            amountMilestone,
-            donationOwnerAddress,
-            0,
-            comment,
-          );
+        this.donateWithBridge(
+          dacId,
+          amountDAC,
+          donationOwnerAddress,
+          amount,
+          comment,
+          allowanceApprovalType,
+        );
+        setTimeout(() => {
```

Comment:
This assumes user will click confirm on his wallet in in one second after first tx popup... so app will show the second popup 1 second after the first one...
It's not good way of implementing it...
The eligible solution is to showing the second popup after user has confirmed the first one.